logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
+      $debugging_event = $event;
+      unset($debugging_event['type']);
```

Comment:
If a single http request is producing more than one log entry - which is very likely to happen - then the extra data should only be logged once, because otherwise it would be redundant. And that single extra log entry should not contain the message and severity of the regular log line, that's why $event gets cloned into $debugging_event which receives the unset's and gets added to the $events array.